junction/core/src/org/littletonrobotics/junction/Logger.java

```diff
@@ -234,18 +234,15 @@ void periodicBeforeUser() {
       // Retrieve new data even if logger is disabled
```

Comment:
This isn't specific to the PR itself but more a general question, why does AKIT retrieve new data if the Logger is disabled? Is it to update instance objects within each Logged thing (such as `MatchDataSender` in `LoggedDriverStation`?